recipes/openvino/all/patches/2023.1.0/0006-tensorflow-proto-odr.patch

```diff
@@ -0,0 +1,746 @@
+rename from src/frontends/tensorflow/src/proto/any.proto
+rename to src/frontends/tensorflow/src/proto/google/protobuf/any.proto
+diff --git a/src/frontends/tensorflow/src/proto/wrappers.proto b/src/frontends/tensorflow/src/proto/google/protobuf/wrappers.proto
+similarity index 100%
+rename from src/frontends/tensorflow/src/proto/wrappers.proto
+rename to src/frontends/tensorflow/src/proto/google/protobuf/wrappers.proto
+diff --git a/src/frontends/tensorflow/src/proto/allocation_description.proto b/src/frontends/tensorflow/src/proto/ov_tensorflow/allocation_description.proto
+similarity index 100%
+rename from src/frontends/tensorflow/src/proto/allocation_description.proto
+rename to src/frontends/tensorflow/src/proto/ov_tensorflow/allocation_description.proto
+diff --git a/src/frontends/tensorflow/src/proto/api_def.proto b/src/frontends/tensorflow/src/proto/ov_tensorflow/api_def.proto
+similarity index 99%
+rename from src/frontends/tensorflow/src/proto/api_def.proto
+rename to src/frontends/tensorflow/src/proto/ov_tensorflow/api_def.proto
+index 810aabc5a2..cbb581973d 100644
+--- a/src/frontends/tensorflow/src/proto/api_def.proto
++++ b/src/frontends/tensorflow/src/proto/ov_tensorflow/api_def.proto
+@@ -21,7 +21,7 @@ option java_outer_classname = "ApiDefProtos";
+ option java_multiple_files = true;
+ option java_package = "org.tensorflow.framework";
+ option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/framework/api_def_go_proto";
+-import "attr_value.proto";
++import "ov_tensorflow/attr_value.proto";
+ 
+ // Used to specify and override the default API & behavior in the
+ // generated code for client languages, from what you would get from
+diff --git a/src/frontends/tensorflow/src/proto/attr_value.proto b/src/frontends/tensorflow/src/proto/ov_tensorflow/attr_value.proto
+similarity index 96%
+rename from src/frontends/tensorflow/src/proto/attr_value.proto
+rename to src/frontends/tensorflow/src/proto/ov_tensorflow/attr_value.proto
+index 3028176c02..b903c30cf9 100644
+--- a/src/frontends/tensorflow/src/proto/attr_value.proto
++++ b/src/frontends/tensorflow/src/proto/ov_tensorflow/attr_value.proto
+@@ -14,9 +14,9 @@ syntax = "proto3";
+ 
+ package tensorflow;
+ 
+-import "tensor.proto";
+-import "tensor_shape.proto";
+-import "types.proto";
++import "ov_tensorflow/tensor.proto";
++import "ov_tensorflow/tensor_shape.proto";
++import "ov_tensorflow/types.proto";
+ 
+ option cc_enable_arenas = true;
+ option java_outer_classname = "AttrValueProtos";
+diff --git a/src/frontends/tensorflow/src/proto/cost_graph.proto b/src/frontends/tensorflow/src/proto/ov_tensorflow/cost_graph.proto
+similarity index 97%
+rename from src/frontends/tensorflow/src/proto/cost_graph.proto
+rename to src/frontends/tensorflow/src/proto/ov_tensorflow/cost_graph.proto
+index dad93a029b..8e4d9788f4 100644
+--- a/src/frontends/tensorflow/src/proto/cost_graph.proto
++++ b/src/frontends/tensorflow/src/proto/ov_tensorflow/cost_graph.proto
```

Comment:
Hi @memsharded 
A few questions regarding Conan patches:
- I observe that I cannot change the same file via patch multiple times - is it expected? Once the first patch is applied, the second one fails.
- In current patch I move files to subfolder, but it does not work and I still see after path applying:
```
[ 85%] Running C++ protocol buffer compiler (protobuf::protoc) on src/frontends/tensorflow/src/proto/reader_base.proto
ov_tensorflow/tensor_shape.proto: File not found.
```
that files are used by old location (`reader_base.proto` must be in `ov_tensorflow` subfolder). Don't you use git to apply patch? the same patches perfectly work in `vcpkg`

Reply:
> Don't you use git to apply patch?

No, patching is done via our own fork of [python-patch-ng](https://github.com/conan-io/python-patch-ng), Conan does not assume git to be available